rewrite-java/src/main/java/org/openrewrite/java/tree/TypeUtils.java

```diff
@@ -137,6 +139,42 @@ public static boolean isOfClassType(@Nullable JavaType type, String fqn) {
         return false;
     }
 
+    /**
+     * @param type The declaring type of the method invocation or constructor.
+     * @param matchOverride Whether to match the {@code Object} type.
+     * @return True if the declaring type matches the criteria of this matcher.
+     */
+    @Incubating(since = "8.1.4")
```

Comment:
We'll perpetually be playing catch up with the new releases, but today we're targetting 8.1.6 ; we can apply this change once the other changes are approved.
```suggestion
    @Incubating(since = "8.1.6")
```